src/mjolnir/ingest_transit.cc

```diff
@@ -607,7 +656,9 @@ void write_shapes(Transit& tile, const tile_transit_info_t& tile_info, feed_cach
 }
 
 // pre-processes feed data and writes to the pbfs (calls the 'write' functions)
-void ingest_tiles(const boost::property_tree::ptree& pt,
+void ingest_tiles(const std::string& gtfs_dir,
+                  const std::string& transit_dir,
+                  const uint32_t pbf_trip_limit,
```

Comment:
pass those into the threads, they had to be generated in the calling function anyways